### ISSUE TITLE
Clarification about ATW

### ISSUE THREAD

**Issue by off99555** [relativty/Relativ#47]
Does this headset support Asynchronous Timewarp (Reprojection)?

**Comment by AdmiraalHugo** [comment]
Hello @off99555 ,

If i'm right, this is build into OSVR, the software we use to connect our headset to SteamVR.

Regards,
Hugo
~ Relativty Committee

**Comment by off99555** [comment]
Yes. OSVR seems to have timewarp feature. You can check it in your headset screen by rotating it fast and see if there are black borders. If there are black borders then it means there is timewarp.
The black borders will appear on the left side of the screen when you rotate to the left. And the black borders will appear on the right side of the screen when you rotate to the right. How big the black border is depends on the latency.
Does your headset have black border when rotating fast?
